doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -127,59 +130,81 @@ Bluetooth samples
 
   * :ref:`direct_test_mode` sample:
 
-    * Fixed handling of the disable Constant Tone Extension command.
-    * The front-end module test parameters are not set to their default value after the DTM reset command.
-    * Added the vendor-specific ``FEM_DEFAULT_PARAMS_SET`` command for restoring the default front-end module parameters.
+    * Updated:
+
+      * The front-end module test parameters are not set to their default value after the DTM reset command.
+      * Added the vendor-specific ``FEM_DEFAULT_PARAMS_SET`` command for restoring the default front-end module parameters.
+
+    * Fixed:
+
+      * Handling of the disable Constant Tone Extension command.
 
   * :ref:`peripheral_hids_mouse` sample:
 
     * Increased the main stack size from 1024 to 1536 bytes.
-    * Increased the stack size of the nRF RPC threads from 1024 to 1280.
+    * Increased the stack size of the nRF RPC threads from 1024 to 1280 bytes.
+
+
+Bluetooth mesh samples
+----------------------
+
+* Updated:
+
+  * All samples to use the Partition Manager, replacing the use of the Device Tree Source flash partitions.
+  * :ref:`bluetooth_mesh_sensor_server` sample:
+
+    * Updated definitions for sensor callbacks to include the :c:struct:`bt_mesh_sensor_srv` context.
```

Comment:
I would rather follow the style from https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases/release-notes-1.8.0.html#bluetooth-mesh-samples and have just one line for 154 and 156.

Reply:
I shall update this if required after our discussion.